### PR TITLE
Patch snowflake-sqlalchemy 1.3.1

### LINKED ISSUE #159: Patch snowflake-sqlalchemy-feedstock to require sqlalchemy >=1.4.0,<2.0.0
https://github.com/conda-forge/snowflake-sqlalchemy-feedstock/pull/8/files#r681172579

- [x] I read [the conda-forge documentation](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge) and could not find the solution for my problem there.

Issue:

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list

```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

```
</details>

### PR DESCRIPTION
**Pull request by janjagusch**
Closes #159 

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by janjagusch** [comment]
<summary>Click to expand diff</summary>
```
Downloading: https://conda.anaconda.org/conda-forge/noarch/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/noarch/repodata.json.bz2
noarch::snowflake-sqlalchemy-1.3.1-pyhd8ed1ab_0.tar.bz2
-    "snowflake-connector-python <3",
-    "sqlalchemy <2"
+    "sqlalchemy >=1.4.0,<2.0.0",
+    "snowflake-connector-python <3.0.0"
Downloading: https://conda.anaconda.org/conda-forge/linux-64/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/linux-64/repodata.json.bz2
linux-64::openpmd-api-0.14.0-mpi_mpich_py36h3243a75_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-64::openpmd-api-0.14.0-mpi_mpich_py37h25e20de_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-64::openpmd-api-0.14.0-mpi_mpich_py37h6adbb96_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-64::openpmd-api-0.14.0-mpi_mpich_py38hb44033b_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-64::openpmd-api-0.14.0-mpi_mpich_py39h597d67e_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-64::openpmd-api-0.14.1-mpi_mpich_py36h3243a75_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-64::openpmd-api-0.14.1-mpi_mpich_py37h25e20de_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-64::openpmd-api-0.14.1-mpi_mpich_py37h6adbb96_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-64::openpmd-api-0.14.1-mpi_mpich_py38hb44033b_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-64::openpmd-api-0.14.1-mpi_mpich_py39h597d67e_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-64::scalapack-2.1.0-h6ba1590_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0"
+    "mpich >=3.4,<4.0.0a0"
Downloading: https://conda.anaconda.org/conda-forge/linux-armv7l/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/linux-armv7l/repodata.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/linux-aarch64/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/linux-aarch64/repodata.json.bz2
linux-aarch64::openpmd-api-0.14.0-mpi_mpich_py36h52081c6_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-aarch64::openpmd-api-0.14.0-mpi_mpich_py37h0f24fb6_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-aarch64::openpmd-api-0.14.0-mpi_mpich_py37hbbaa06b_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-aarch64::openpmd-api-0.14.0-mpi_mpich_py38hbfb59e0_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-aarch64::openpmd-api-0.14.0-mpi_mpich_py39hbd1f846_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-aarch64::scalapack-2.1.0-h478e2f9_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0"
+    "mpich >=3.4,<4.0.0a0"
Downloading: https://conda.anaconda.org/conda-forge/linux-ppc64le/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/linux-ppc64le/repodata.json.bz2
linux-ppc64le::openpmd-api-0.14.0-mpi_mpich_py36hee51da2_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-ppc64le::openpmd-api-0.14.0-mpi_mpich_py37h334d0ee_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-ppc64le::openpmd-api-0.14.0-mpi_mpich_py37hfa7fdc2_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-ppc64le::openpmd-api-0.14.0-mpi_mpich_py38h78c6e08_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-ppc64le::openpmd-api-0.14.0-mpi_mpich_py39h2d22980_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-ppc64le::openpmd-api-0.14.1-mpi_mpich_py36hee51da2_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-ppc64le::openpmd-api-0.14.1-mpi_mpich_py37h334d0ee_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-ppc64le::openpmd-api-0.14.1-mpi_mpich_py37hfa7fdc2_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-ppc64le::openpmd-api-0.14.1-mpi_mpich_py38h78c6e08_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-ppc64le::openpmd-api-0.14.1-mpi_mpich_py39h2d22980_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-ppc64le::scalapack-2.1.0-hb27407b_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0"
+    "mpich >=3.4,<4.0.0a0"
Downloading: https://conda.anaconda.org/conda-forge/osx-64/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/osx-64/repodata.json.bz2
osx-64::openpmd-api-0.14.0-mpi_mpich_py36hafb1288_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
osx-64::openpmd-api-0.14.0-mpi_mpich_py37hc9a823c_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
osx-64::openpmd-api-0.14.0-mpi_mpich_py37hcb5fae7_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
osx-64::openpmd-api-0.14.0-mpi_mpich_py38h6b59ff8_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
osx-64::openpmd-api-0.14.0-mpi_mpich_py39he645e96_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
osx-64::openpmd-api-0.14.1-mpi_mpich_py36hafb1288_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
osx-64::openpmd-api-0.14.1-mpi_mpich_py37hc9a823c_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
osx-64::openpmd-api-0.14.1-mpi_mpich_py37hcb5fae7_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
osx-64::openpmd-api-0.14.1-mpi_mpich_py38h6b59ff8_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
osx-64::openpmd-api-0.14.1-mpi_mpich_py39he645e96_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
osx-64::scalapack-2.1.0-hb62163f_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0"
+    "mpich >=3.4,<4.0.0a0"
Downloading: https://conda.anaconda.org/conda-forge/osx-arm64/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/osx-arm64/repodata.json.bz2
osx-arm64::openpmd-api-0.14.0-mpi_mpich_py38h00ff84a_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
osx-arm64::openpmd-api-0.14.0-mpi_mpich_py39hc5352e2_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
osx-arm64::openpmd-api-0.14.1-mpi_mpich_py38h00ff84a_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
osx-arm64::openpmd-api-0.14.1-mpi_mpich_py39hc5352e2_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
Downloading: https://conda.anaconda.org/conda-forge/win-32/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/win-32/repodata.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/win-64/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/win-64/repodata.json.bz2
```
</details>

File: recipe/gen_patch_json.py [hunk 1]
```diff
@@ -1061,6 +1061,13 @@ def _gen_new_index(repodata, subdir):
                 i = record["depends"].index("py-tools-ds >=0.15.10")
                 record["depends"][i] = "py-tools-ds >=0.15.10,<=0.15.11"
 
+        # https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/issues/159
+        if record_name == "snowflake-sqlalchemy" and record["version"] == "1.3.1" and record["build_number"] == 0:
```
Comment (xhochy) [inline recipe/gen_patch_json.py:1065]: Have a look at the section above. That one is a bit neater as it doesn't copy the list of dependencies but just replaces the offending entry.

**Review by xhochy — COMMENTED** [review]
Please also post the output of `show_diff`

File: recipe/gen_patch_json.py (hunk 1 above)
Comment (janjagusch) [reply recipe/gen_patch_json.py:1065]: Done.

**Comment by jtilly** [comment]
The same issue exists for `snowflake-sqlalchemy 1.2.5`.

**Comment by janjagusch** [comment]
> Please also post the output of `show_diff`

I have already done that [here](https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/160#issuecomment-893301425)

**Comment by janjagusch** [comment]
<details>
<summary>Click to expand diff</summary>
```
Downloading: https://conda.anaconda.org/conda-forge/noarch/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/noarch/repodata.json.bz2
noarch::snowflake-sqlalchemy-1.2.5-pyhd8ed1ab_0.tar.bz2
-    "snowflake-connector-python <3",
-    "sqlalchemy <2"
+    "snowflake-connector-python <3.0.0",
+    "sqlalchemy >=1.4.0,<2.0.0"
noarch::snowflake-sqlalchemy-1.3.1-pyhd8ed1ab_0.tar.bz2
-    "snowflake-connector-python <3",
-    "sqlalchemy <2"
+    "snowflake-connector-python <3.0.0",
+    "sqlalchemy >=1.4.0,<2.0.0"
Downloading: https://conda.anaconda.org/conda-forge/linux-64/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/linux-64/repodata.json.bz2
linux-64::openpmd-api-0.14.0-mpi_mpich_py36h3243a75_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-64::openpmd-api-0.14.0-mpi_mpich_py37h25e20de_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-64::openpmd-api-0.14.0-mpi_mpich_py37h6adbb96_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-64::openpmd-api-0.14.0-mpi_mpich_py38hb44033b_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-64::openpmd-api-0.14.0-mpi_mpich_py39h597d67e_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-64::openpmd-api-0.14.1-mpi_mpich_py36h3243a75_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-64::openpmd-api-0.14.1-mpi_mpich_py37h25e20de_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-64::openpmd-api-0.14.1-mpi_mpich_py37h6adbb96_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-64::openpmd-api-0.14.1-mpi_mpich_py38hb44033b_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-64::openpmd-api-0.14.1-mpi_mpich_py39h597d67e_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-64::scalapack-2.1.0-h6ba1590_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0"
+    "mpich >=3.4,<4.0.0a0"
Downloading: https://conda.anaconda.org/conda-forge/linux-armv7l/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/linux-armv7l/repodata.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/linux-aarch64/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/linux-aarch64/repodata.json.bz2
linux-aarch64::openpmd-api-0.14.0-mpi_mpich_py36h52081c6_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-aarch64::openpmd-api-0.14.0-mpi_mpich_py37h0f24fb6_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-aarch64::openpmd-api-0.14.0-mpi_mpich_py37hbbaa06b_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-aarch64::openpmd-api-0.14.0-mpi_mpich_py38hbfb59e0_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-aarch64::openpmd-api-0.14.0-mpi_mpich_py39hbd1f846_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-aarch64::openpmd-api-0.14.1-mpi_mpich_py36h52081c6_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-aarch64::openpmd-api-0.14.1-mpi_mpich_py37h0f24fb6_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-aarch64::openpmd-api-0.14.1-mpi_mpich_py37hbbaa06b_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-aarch64::openpmd-api-0.14.1-mpi_mpich_py38hbfb59e0_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-aarch64::scalapack-2.1.0-h478e2f9_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0"
+    "mpich >=3.4,<4.0.0a0"
Downloading: https://conda.anaconda.org/conda-forge/linux-ppc64le/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/linux-ppc64le/repodata.json.bz2
linux-ppc64le::openpmd-api-0.14.0-mpi_mpich_py36hee51da2_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-ppc64le::openpmd-api-0.14.0-mpi_mpich_py37h334d0ee_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-ppc64le::openpmd-api-0.14.0-mpi_mpich_py37hfa7fdc2_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-ppc64le::openpmd-api-0.14.0-mpi_mpich_py38h78c6e08_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-ppc64le::openpmd-api-0.14.0-mpi_mpich_py39h2d22980_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-ppc64le::openpmd-api-0.14.1-mpi_mpich_py36hee51da2_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-ppc64le::openpmd-api-0.14.1-mpi_mpich_py37h334d0ee_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-ppc64le::openpmd-api-0.14.1-mpi_mpich_py37hfa7fdc2_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-ppc64le::openpmd-api-0.14.1-mpi_mpich_py38h78c6e08_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-ppc64le::openpmd-api-0.14.1-mpi_mpich_py39h2d22980_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
linux-ppc64le::scalapack-2.1.0-hb27407b_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0"
+    "mpich >=3.4,<4.0.0a0"
Downloading: https://conda.anaconda.org/conda-forge/osx-64/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/osx-64/repodata.json.bz2
osx-64::openpmd-api-0.14.0-mpi_mpich_py36hafb1288_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
osx-64::openpmd-api-0.14.0-mpi_mpich_py37hc9a823c_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
osx-64::openpmd-api-0.14.0-mpi_mpich_py37hcb5fae7_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
osx-64::openpmd-api-0.14.0-mpi_mpich_py38h6b59ff8_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
osx-64::openpmd-api-0.14.0-mpi_mpich_py39he645e96_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
osx-64::openpmd-api-0.14.1-mpi_mpich_py36hafb1288_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
osx-64::openpmd-api-0.14.1-mpi_mpich_py37hc9a823c_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
osx-64::openpmd-api-0.14.1-mpi_mpich_py37hcb5fae7_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
osx-64::openpmd-api-0.14.1-mpi_mpich_py38h6b59ff8_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
osx-64::openpmd-api-0.14.1-mpi_mpich_py39he645e96_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
osx-64::scalapack-2.1.0-hb62163f_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0"
+    "mpich >=3.4,<4.0.0a0"
Downloading: https://conda.anaconda.org/conda-forge/osx-arm64/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/osx-arm64/repodata.json.bz2
osx-arm64::openpmd-api-0.14.0-mpi_mpich_py38h00ff84a_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
osx-arm64::openpmd-api-0.14.0-mpi_mpich_py39hc5352e2_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
osx-arm64::openpmd-api-0.14.1-mpi_mpich_py38h00ff84a_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
osx-arm64::openpmd-api-0.14.1-mpi_mpich_py39hc5352e2_0.tar.bz2
-    "mpich >=3.4,<3.5.0a0",
+    "mpich >=3.4,<4.0.0a0",
Downloading: https://conda.anaconda.org/conda-forge/win-32/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/win-32/repodata.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/win-64/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/win-64/repodata.json.bz2
```
</details>

**Pull request merged**